testing/test_cacheprovider.py

```diff
@@ -14,6 +14,17 @@
 pytest_plugins = ("pytester",)
 
 
+@pytest.fixture(scope="module", autouse=True)
+def handle_env():
+    """Ensure env is like most of the tests expect it, i.e. not using tox."""
+    orig_env = os.environ.pop("TOX_ENV_DIR", None)
```

Comment:
This could also easily be written as:

```python
@pytest.fixture(scope="module", autouse=True)
def handle_env(monkeypatch):
    monkeypatch.delenv('TOX_ENV_DIR', raising=False)
```

Reply:
Nope, ScopeMismatch.

Reply:
Oh true, but in that case I would use `"function"` scope as the cost of patching for each test is negligible.

Reply:
I see, and used that initially IIRC, but then figured that it is at least something, if only for when stepping through hook callers via pdb.. ;)